pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -133,7 +133,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 			Rules: []authorizationapi.PolicyRule{
-				authorizationapi.NewRule("impersonate").Groups(kapiGroup).Resources(authorizationapi.SystemUserResource).Names(SystemAdminUsername).RuleOrDie(),
+				authorizationapi.NewRule("impersonate").Groups(authzGroup, legacyAuthzGroup, userGroup).Resources(authorizationapi.SystemUserResource).Names(SystemAdminUsername).RuleOrDie(),
```

Comment:
> @deads2k PTAL (i think authzGroup is not required, what fixed this was addition of userGroup)

Our system users kind should be grouped, so the new code is probably overly loose.